_test/test_transformation/test_spher_proj.m

```diff
@@ -27,7 +27,7 @@ function test_coord_transf_PixData_plus_offset(~)
         end
 
         function test_coord_transf_4D_plus_offset(~)
-            proj = spher_proj();
+            proj = spher_proj('alatt',2*pi,'angdeg',90);
```

Comment:
Does it make sense for `spher_proj` to have an `alatt` given it's not on the lattice?

Reply:
There is complex problem related to offset.  I've defined it in `hkl`. There is `img_offset` defined in projection units. 
For spherical projection it is probably reasonable to define offset in `hkl` as you move your projection into the Brulluoin zone of interest.  There is `img_offset` defined in the projection units. 

as soon as this definition is there,  I need the lattice parameters to transform one into another. 
May be it is incorrect decision. I will be discussing this with Toby. If the decision is taken, it would be separate ticket to resolve the issue (for all projections, not spherical only)